src/functions-reference/unbounded_discrete_distributions.Rmd

```diff
@@ -219,29 +219,73 @@ dropping constant additive terms.
 
 ### Stan Functions
 
+<!-- real; neg_binomial_2_log_glm_lpmf; (int y | matrix x, real alpha, vector beta, real phi); -->
+\index{{\tt \bfseries neg\_binomial\_2\_log\_glm\_lpmf }!{\tt (int y \textbar\ matrix x, real alpha, vector beta, real phi): real}|hyperpage}
+
+`real` **`neg_binomial_2_log_glm_lpmf`**`(int y | matrix x, real alpha, vector beta, real phi)`<br>\newline
+The log negative binomial probability mass of `y` given log-location
+`alpha + x * beta` and inverse overdispersion control `phi`, where a
+constant intercept `alpha`, `phi` and dependant variable `y` are used for all observations. 
```

Comment:
`alpha` doesn't have to be constant---it can be a parameter.  It's just a single real value that's reused.  I'd suggest

```
and inverse overdispersion parameter `phi`.  The intercept `alpha`, overdispersion parameter
`phi`, and dependent variable `y` are used for all observations.
```

Reply:
fixed